mesonbuild/interpreter.py

```diff
@@ -3627,22 +3627,36 @@ def func_build_target(self, node, args, kwargs):
 
     @permittedKwargs(permitted_kwargs['vcs_tag'])
     def func_vcs_tag(self, node, args, kwargs):
+        for k in list(kwargs):
+            v = kwargs[k]
+            if is_arg_disabled(v):
+                if k == 'command' and not is_arg_disabled(kwargs.get('fallback', Disabler())):
+                    kwargs['command'] = Disabler() # Use fallback when command is a disabler
```

Comment:
Is not this already a disabler?

Reply:
it could be a list containing a disabler